components/layout/Layout.tsx

```diff
@@ -12,15 +12,15 @@ interface LayoutProps {
   children: any[]
   color?: 'white' | 'secondary' | 'seafoam'
   sourceProviderConnection: any
-  editMode: boolean
+  preview: boolean
 }
 
 export const Layout = styled(
   ({
     children,
     color,
     sourceProviderConnection,
```

Comment:
I bet this can be removed

Reply:
Just checked. `sourceProviderConnection` is only being used to passed to the CreateBlogPostPlugin. That plugin no longer needs `sourceProviderConnection` so we can remove all these references 🙌